src/mmw/js/src/modeling/views.js

```diff
@@ -654,6 +644,11 @@ var ResultsTabPanelView = Marionette.ItemView.extend({
     template: resultsTabPanelTmpl,
     attributes: {
         role: 'presentation'
+    },
+    initialize: function() {
+        this.listenTo(this.model, 'change:polling', function() {
```

Comment:
I think this could just be `this.listenTo(this.model, 'change:polling', this.render)`.